com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -137,25 +152,44 @@ public int AddEntry(in VariableKey k)
             return pos;
         }
 
-        internal void AddSpawn(SnapshotSpawnCommand command)
+        internal List<ulong> GetClientList()
         {
-            if (NumSpawns < k_MaxSpawns)
+            List<ulong> clientList;
+            clientList = new List<ulong>();
+
+            if (!NetworkManager.IsServer)
             {
-                command.TargetClientIds = new List<ulong>();
-                if (!m_NetworkManager.IsServer)
-                {
-                    command.TargetClientIds.Add(m_NetworkManager.ServerClientId);
-                }
-                else
+                clientList.Add(NetworkManager.ServerClientId);
+            }
+            else
+            {
+                foreach (var clientId in NetworkManager.ConnectedClientsIds)
                 {
-                    foreach (var clientId in m_NetworkManager.ConnectedClientsIds)
+                    if (clientId != NetworkManager.ServerClientId)
                     {
-                        if (clientId != m_NetworkManager.ServerClientId)
-                        {
-                            command.TargetClientIds.Add(clientId);
-                        }
+                        clientList.Add(clientId);
                     }
                 }
+            }
+
+            return clientList;
+        }
+
+        internal void AddSpawn(SnapshotSpawnCommand command)
+        {
+            if (NumSpawns >= m_MaxSpawns)
+            {
+                Array.Resize(ref Spawns, 2 * m_MaxSpawns);
+                m_MaxSpawns = m_MaxSpawns * 2;
+                Debug.Log($"[JEFF] spawn size is now {m_MaxSpawns}");
```

Comment:
[MATT] Did you mean to leave this in?   :)

Reply:
Actually, yes. But if it is an issue, I'm willing to remove it. The release will not be cut just yet, and even if it were, this is not used, by default. Even it it were, this log would only occur every time the buffer doubles, so, at most 4-5 times ?

Otherwise, with the number of merges we do, I found I was wasting tons of time adding/removing these. Your call.